backend/.gitignore

```diff
@@ -1,18 +1,22 @@
-# Byte-compiled / optimized / DLL files
-__pycache__/
```

Comment:
These should not be removed from the .gitignore. we do not want to commit pycache files.